docker/build_image.sh

```diff
@@ -9,6 +9,7 @@ BASE_IMAGE="ubuntu:18.04"
 CUSTOM_TAG=false
 CUDA_VERSION=""
 USE_LOCAL_SERVE_FOLDER=false
+BUILD_WITH_IPEX=false
```

Comment:
NIT: Consider adding a check to make sure `-g` and `-ipex` are both not set

Reply:
Hi, have made the changes.